### PR TITLE
Quiet build warnings in bigput test

### PR DESCRIPTION
**Pull request by jdinan**

### CODE REVIEW

File: config/ax_pthread.m4 [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+#
+#   Updated for Autoconf 2.68 by Daniel Richard G.
+#
+# LICENSE
+#
+#   Copyright (c) 2008 Steven G. Johnson <stevenj@alum.mit.edu>
+#   Copyright (c) 2011 Daniel Richard G. <skunk@iSKUNK.ORG>
+#
+#   This program is free software: you can redistribute it and/or modify it
+#   under the terms of the GNU General Public License as published by the
```
Comment (kseager) [inline config/ax_pthread.m4]: GPL!?!

Comment (jdinan) [inline config/ax_pthread.m4]: Oh, crap.  Good catch.  The other m4 script I pulled from this website was BSD compatible; I forgot to check this one.  :(

Comment (jdinan) [inline config/ax_pthread.m4]: I don't think the exception helps.  I'll need to pitch this macro and find some other way to detect pthreads support.

Comment (kseager) [inline config/ax_pthread.m4]: ok thanks!

**Comment by jdinan** [comment]
Removed the pthreads autoconf changes from this PR.  Should be fine to merge now.

**Pull request merged**